tests/test_conversion.py

```diff
@@ -0,0 +1,92 @@
+from magicgui import magic_factory
+from napari_plugin_engine import napari_hook_implementation
```

Comment:
```suggestion
from importlib.metadata import PackageNotFoundError
import pytest
from magicgui import magic_factory
from napari_plugin_engine import napari_hook_implementation
```
For a test I'm suggesting below.